### ISSUE TITLE
docs: list of demo cases/real-life success stories

### ISSUE THREAD

**Issue by yarikoptic** [iterative/dvc.org#1749]
Looking at https://dvc.org/doc/use-cases/versioning-data-and-model-files/tutorial I wanted to look at some real use cases which would demonstrate the power and features of DVC.  I wondered if there is some examples (on github?) you could point to?

Thank you in advance!

**Comment by jorgeorpinel** [comment]
Hi Yaroslav,

We only maintain basic guides in our docs because DVC is still evolving fast. Other than that versioning tutorial you found, this is the main one: https://dvc.org/doc/start (it comes with a matching GitHub repo)

There are several other DVC tutorials published in well-respected blogs both by us and by 3rd-party contributors, which can be found via web searches. Many of them have matching repos as well. Here are a few: 

- TRP: https://realpython.com/python-data-version-control/
- TDS: https://towardsdatascience.com/version-control-ml-model-4adb2db5f87c and https://towardsdatascience.com/how-to-use-data-version-control-dvc-in-a-machine-learning-project-a78245c0185
- https://blog.codecentric.de/en/2019/03/walkthrough-dvc/
- https://medium.com/qonto-engineering/using-dvc-to-create-an-efficient-version-control-system-for-data-projects-96efd94355fe
- DVC Blog: https://dvc.org/blog/a-public-reddit-dataset

We're also producing short videos now, which you can find here: https://www.youtube.com/channel/UC37rp97Go-xIX3aNFVHhXfQ/videos

**Comment by jorgeorpinel** [comment]
P.s. feel free to reach us directly at dvc.org/chat with other questions 🙂

**Comment by shcheklein** [comment]
@yarikoptic yeah, we never actually spent time to find and track existing public repos (I guess it's even our goal, majority of cases are still private repos), neither doing actual "user stories" besides what @jorgeorpinel already mentioned and some talks that are publicly available. 

Are you curious in general or in some specific aspect? I might remember and point you to some

**Comment by yarikoptic** [comment]
Thank you @jorgeorpinel for the references, I will check them out!

@shcheklein yeap -- having "user stories" and some real world examples referenced within docs or even some dedicated registry of those -- would have been great resource to all dvc users and newcomers.  
In my case I just wanted to find some real world examples on how people use dvc, so to appreciate features dvc provides and which are actually used in real-life "workflows", so we could adequately compare DataLad to DVC (what we cannot do in DataLad and what DVC can do; or how we could possibly do in DataLad what DVC could do).

**Comment by shcheklein** [comment]
@yarikoptic yep, I feel you! I have the same problems with the other tools (I haven't checked the DataLad website for a while though). Don't know even what to recommend here ... if you can use my time we jump on a call and sync, discuss things.

**Comment by jorgeorpinel** [comment]
> real world examples on how people use dvc

This is a great idea for the website though. Something between "who is using it/ success stories" and "real-life tutorials".

**Comment by yarikoptic** [comment]
https://dagshub.com uses/integrates-with DVC, so I guess some nice examples could come from it or could be shared/visualized on it.  a "sync" -- would be nice.  Need to un-bury myself from ongoing undertakings a bit first.
Re datalad website -- better go straight to  http://handbook.datalad.org/  ;)

**Comment by shcheklein** [comment]
@jorgeorpinel Ideally we can keep some links in the use cases to these "success stories" I think.

**Comment by jorgeorpinel** [comment]
Yeah but I was thinking something outside of /doc more like "who is using it (and how in our case)" or "case studies" e.g.:

![image](https://user-images.githubusercontent.com/1477535/91906205-69de7b00-ec6d-11ea-930e-ccc67f9dbe47.png)

**Comment by jorgeorpinel** [comment]
p.s. I also think use cases could belong outside of /doc so in that case yes, we could combine them to keep things simple.

**Comment by yarikoptic** [comment]
> In my case I just wanted to find some real world examples on how people use dvc, so to appreciate features dvc provides and which are actually used in real-life "workflows", so we could adequately compare DataLad to DVC (what we cannot do in DataLad and what DVC can do; or how we could possibly do in DataLad what DVC could do).

And here comes "DataLad as DVC for ML analyses": you are welcome to read [the rendered document](https://datalad-handbook--569.org.readthedocs.build/en/569/beyond_basics/101-168-dvc.html) and/or provide comments/suggestions/(some)rebuttal to https://github.com/datalad-handbook/book/pull/569 . I hope it might be useful not only for datalad folks but also for DVC users and possibly developers ;)
